desdeo_tools/utilities/model_management.py

```diff
@@ -0,0 +1,85 @@
+from numpy.core.numeric import indices
+from numpy.lib.arraysetops import unique
+from desdeo_tools.scalarization.ASF import SimpleASF
+from numba import njit
+import numpy as np
+from desdeo_problem.problem.Objective import ScalarDataObjective
+import pandas as pd
+import copy
+from desdeo_problem.surrogatemodels.SurrogateModels import GaussianProcessRegressor
```

Comment:
desdeo-tools should not depend on methods from desdeo-problem. If you need to use code from both repositories, implement these algorithms in desdeo-emo.